docs/security/trust/content_trust.md

```diff
@@ -267,13 +268,18 @@ Because the tag `docker/cliffs:latest` is not trusted, the `pull` fails.
 A user who wants to disable content trust for a particular operation can use the
```

Comment:
I wonder if this section should be combined with the above section about passing `--disable-content-trust`.  Since this section describes doing  `docker push --disable-content-trust <username>/trusttest:latest`, that would fix the issue with https://github.com/docker/docker/pull/23513/files#r67012847.